Code/GraphMol/Wrap/rough_test.py

```diff
@@ -5652,6 +5652,91 @@ def testSetBondDirFromStereo(self):
     self.assertEqual(m1.GetBondWithIdx(0).GetBondDir(),Chem.BondDir.ENDUPRIGHT)
     self.assertEqual(m1.GetBondWithIdx(2).GetBondDir(),Chem.BondDir.ENDDOWNRIGHT)
 
+  def testAssignChiralTypesFromMolParity(self):
```

Comment:
It doesn't seem like this test needs to be this complicated. Can't you just test using the fact that you know what the answer should be based on the contents of the mol file?

Reply:
Not sure I agree. The test tries all permutations of bonds while keeping atom order constant, so it actually thoroughly checks that the swap logic works. It may be a bit overkill, but it is quick. Do you really hate it?

Reply:
I don’t hate it, but it’s definitely not trivial to see what the test is actually doing
No need to change if you prefer this approach

Reply:
I don’t hate it, but it’s definitely not trivial to see what the test is actually doing
No need to change if you prefer this approach